aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -536,12 +536,21 @@ Resources:
                   - 'rds:RestoreDBClusterToPointInTime'
                   - 'rds:DeleteDBCluster'
```

Comment:
Remove this since it's broken out below.

Reply:
Oops, yes.  That was the whole point of breaking out Policy.  Done!